data_driven_onboarding/data-driven-onboarding.sql

```diff
@@ -145,11 +169,14 @@ select
 
     -- average wait time for repairs after a landlord has been notified of a violation. for the the entire associated portfolio
     -- may return null if unknown
-    P.AverageWaitTimeForPortfolio as average_wait_time_for_repairs_for_portfolio
+    P.AverageWaitTimeForPortfolio as average_wait_time_for_repairs_for_portfolio,
+    M.majorcategory as Most_Common_Category_Of_HPD_Complaint,
```

Comment:
Can you add a comment documenting this?  A list of the categories, or a link to the data description that explains them, might be particularly useful.

Also, can you make the variable name be all-lowercase like the other ones?

Reply:
just made some commits with changes that you requested!